ImGuiStyler/Alignment.cs

```diff
@@ -4,14 +4,29 @@ namespace ktsu.ImGuiStyler;
 using ImGuiNET;
 using ktsu.ScopedAction;
 
+/// <summary>
+/// Provides methods for aligning content within a container.
+/// </summary>
 public static class Alignment
 {
+	/// <summary>
+	/// Centers the content within the available content region.
+	/// </summary>
+	/// <param name="contentSize">The size of the content to be centered.</param>
 	public class Center(Vector2 contentSize) : CenterWithin(contentSize, new Vector2(ImGui.GetContentRegionAvail().X, contentSize.Y))
 	{
 	}
 
+	/// <summary>
+	/// Centers the content within a specified container size.
```

Comment:
Should this mention how the cursor is advanced?

Reply:
See: https://github.com/ktsu-dev/ImGuiStyler/pull/68#discussion_r1879304508